tinyssh_install

```diff
@@ -1,14 +1,27 @@
 #!/bin/bash
 
+KEY_DIR=/etc/tinyssh/sshkeydir
+
+error_no_keys() {
+  echo "ERROR: There are no keys in ${KEY_DIR}! Something must have gone wrong. Please debug $(realpath $0) to fix it."
+}
+
+verify_keys() {
+  if [ ! -d "$KEY_DIR" -o -n "$(find "$KEY_DIR" -maxdepth 0 -empty)" ]; then
```

Comment:
This tests whether `$KEY_DIR` exists and is non-empty. Perhaps it would be better to get the information directly from `tinysshd-printkey` if it handles errors meaningfully.